build.gradle

```diff
@@ -1,7 +1,25 @@
+buildscript {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+    }
+
+}
+
+plugins {
+    id 'maven-publish'
+}
+
 ext {
     flinkVersion = System.getenv('flinkVersion') ?: '1.14.3'
 }
 
+def pyZipVersion = '0.0.1'
```

Comment:
this should come from version.txt

Reply:
Changed pyZipVersion to take value from version.txt